.npmrc

```diff
@@ -0,0 +1 @@
+@addepar:registry = "https://registry.npmjs.org/"
```

Comment:
do we need these files?

Reply:
Yes, otherwise it will use our private registry in the yarn.lock file for all the `@addepar/` packages